addons/account/tests/common.py

```diff
@@ -5,6 +5,7 @@
 from odoo.tools.float_utils import float_round
 
 from odoo.addons.product.tests.common import ProductCommon
+from odoo.addons.stock.tests.common import TestStockCommonMixin
```

Comment:
avoid to use * in the commit title. Target the main module. In this case stock
[IMP] stock: create warehouse automatically only for the first company